README.md

```diff
@@ -1,6 +1,10 @@
 # Poolgirl - A sexy Erlang worker pool factory
 
 [![](https://travis-ci.com/MiniclipPortugal/poolgirl.png?branch=master)](https://travis-ci.com/MiniclipPortugal/poolgirl)
+[![Erlang][erlang badge]][erlang downloads]
+
+[erlang badge]: https://img.shields.io/badge/erlang-%3E%3D19.0-lightgrey
```

Comment:
^ Very cool, didn't know about these.

Reply:
I think I found one of these in `elli-lib`.